cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -79,12 +79,16 @@
             "description": "The affiliation of the author",
             "type": "string"
           },
+          "rorid": {
+            "description": "The ROR ID for the author's affiliation",
```

Comment:
Shall we add a comment at the end of the description like: "Multiple affiliations are separated by semicolon (`;`)." both in `affiliation` and `rorid`?

(If you change this, you could also rebase against latest `master` for merging...)